Microsoft/WebTemplateStudio
# Header component should be NavBar component

**Summary:**
rename HeaderComponent and app-header to **NavBarComponent** and **app-nav-bar**

Suggestion from experts #681:
"The class name HeaderComponent and component selector app-header don't match. It should be: NavBarComponent for the class name and app-nav-bar for the component selector (or any other name, but they have to be consistent)"

**How to test:**
1. generate angular project with at least 1 page 
2. start the project, it works 
3. you should see NavBarComponent in generated file /src/app/app.module.ts
